### ISSUE TITLE
WidgetBot.io not successfully invoking SparksieBOT commands

### ISSUE THREAD

**Issue by spark-c** [spark-c/spark-c.github.io#7]
It seems that either something with WidgetBot has changed, perhaps there is some difference with the way I am "logging in" on my end on the site.

Either way, SparksieBOT is no longer listening to commands sent via WidgetBOT. Strangely, SB still responds correctly to "on_message" events even though they are sent by the same "user" as the ignored command attempts.

It is possible that the issue is with SB. If so, it is likely a problem with the way that WidgetBot presents its "users". SB checks to see if the bot sending a command is WidgetBot, and responds if so. Perhaps WidgetBot is somehow adjusting its identity based upon the user it is representing?

TL;DR - SparksieBOT will respond to commands invoked via Discord, however does not respond to commands invoked via WidgetBot from this portfolio site.
